replica/distributed_loader.cc

```diff
@@ -6,6 +6,7 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
I'm confused by this commit. Does it contain two commits squashed?

Reply:
The coroutinization one? yes.
I squashed that and the indentation patch following together

Reply:
Yes please.

Reply:
done in v2